src/components/Goals/BikeGoal/DateSelector/helpers.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+
+// TODO: remove eslint-disable by convert 'src/lib/timeseries' in TS
+// or at least getEarliestTimeserie
+
+import type { TimeSeries } from 'cozy-client/types/models/timeseries'
+
+import differenceInCalendarYears from 'date-fns/differenceInCalendarYears'
+import subYears from 'date-fns/subYears'
+
+import { getEarliestTimeserie } from 'src/lib/timeseries'
+
+export const makeMenuDates = (timeseries: Array<TimeSeries>): Array<string> => {
```

Comment:
nit: `Array<Type>` can be written `Type[]` which is a bit less verbose

Reply:
oui je sais, j'ai mis ça intentionnellement car il n'y a pas ce genre de raccourcis pour tous les types, histoire d'avoir toujours la même écriture...